kubernetes/resource_kubernetes_persistent_volume_claim_test.go

```diff
@@ -716,11 +716,6 @@ resource "kubernetes_persistent_volume" "test" {
       }
     }
   }
-  lifecycle {
-    ignore_changes = [
-      spec[0].claim_ref,
-    ]
-  }
 }
```

Comment:
This section was causing the test to pass, when it shouldn't have. It masked the perpetual diff.